cms/static/js/models/settings/course_details.js

```diff
@@ -62,6 +67,15 @@ var CourseDetails = Backbone.Model.extend({
         // NOTE don't return empty errors as that will be interpreted as an error state
     },
 
+    convertDateStringsToObjects: function(attrs){
```

Comment:
This seems like a pretty useful utility function -- maybe we can move it into one of the helper libraries?